src/base/memory_monitor_cl.cpp

```diff
@@ -0,0 +1,160 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * memory_monitor_cl.cpp - definitions of client functions
+ *                         for memory monitoring module
+ */
+
+#include <cstring>
+#include <string>
+#include <stdio.h>
+
+#include "memory_monitor_cl.hpp"
+
+/*
+ * mmon_convert_module_name_to_index -
+ *
+ * return: module index
+ *
+ *   module_name(in):
+ */
+int
+mmon_convert_module_name_to_index (const char *module_name)
+{
+  for (int i = 0; i < MMON_MODULE_LAST; i++)
+    {
+      if (!strcmp (module_name, module_names[i]))
+	{
+	  return i;
+	}
+    }
+
+  /* for code protection. unreachable */
+  return -1;
+}
+
+/*
+ * mmon_print_server_info -
+ *
+ * return:
+ *
+ *   server_info(in/out):
+ */
+void
+mmon_print_server_info (MMON_SERVER_INFO &server_info)
+{
+  fprintf (stdout, "====================Cubrid Memmon====================\n");
+  fprintf (stdout, "Server Name: %s\n", server_info.name);
+  fprintf (stdout, "Total Memory Usage(KB): %lu\n\n", CONVERT_TO_KB_SIZE (server_info.total_mem_usage));
+  fprintf (stdout, "-----------------------------------------------------\n");
+}
+
+/*
+ * mmon_print_module_info -
+ *
+ * return:
+ *
+ *   module_info(in):
+ */
+void
+mmon_print_module_info (std::vector<MMON_MODULE_INFO> &module_info)
+{
+  const std::string init_size_str ("Init Size(KB)");
```

Comment:
Nit.
```suggestion
  const std::string init_size_str = "Init Size(KB)";
```

Reply:
And I'd use `auto` following https://herbsutter.com/2013/08/12/gotw-94-solution-aaa-style-almost-always-auto/:
```suggestion
  const auto init_size_str = std::string {"Init Size(KB)"};
```

Reply:
Thanks. Done.